frontend/src/Pages/StudentProfile.tsx

```diff
@@ -0,0 +1,287 @@
+                        </div>
+                        {/* Chart */}
+                        <div className="flex-1 h-[240px] flex flex-col gap-4">
+                            <div className="card card-row-padding overflow-hidden">
+                                <h1 className="">
+                                    {' '}
+                                    {/* {metrics?.session_engagement.user_info.name_first +
+                                        " 's recent Activity"} */}
+                                </h1>
+                                <div className=" items-stretch">
+                                    <div className="h-[240px] overflow-visible">
+                                        <ResponsiveContainer
+                                            className="h-full p-7"
+                                            width="100%"
+                                            height="100%"
+                                            debounce={500}
+                                        >
+                                            <EngagementRateGraph
+                                                data={
+                                                    metrics?.session_engagement
+                                                        ?.user_engagement_times ??
+                                                    []
+                                                }
+                                                viewType="daily"
+                                            />
+                                        </ResponsiveContainer>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    {/* Cards */}
+                    <div className="w-[1/2] grid grid-cols-3 gap-4 mb-6 mt-6">
+                        <StatsCard
+                            title="Days Active"
+                            number={metrics.activity_engagement.total_active_days_monthly.toFixed(
+                                0
+                            )}
+                            label="Days Active This Month"
+                            tooltip="Total number of days resident has been active in UnlockedEd"
+                            useToLocaleString={false}
+                        />
+                        <StatsCard
+                            title="Average Activity Time"
+                            number={avgNumber}
+                            label={avgLabel}
+                            tooltip={avgToolTip}
+                            useToLocaleString={false}
+                        />
+                        <StatsCard
+                            title="Total Hours"
+                            number={weekNumber}
+                            label={weekLabel}
+                            tooltip={weekToolTip}
+                        />
```

Comment:
Can we make the following changes:
- Title: Total Hours This Week
- Label: Hours
- Value: 0 if the value is 0, < 1 if the value is greater than 0 minutes but less than 1 hour
- Tooltip is "Total hours logged in to UnlockEd"